### PR TITLE
Add "Hello Kedro" example to examples repo

### PR DESCRIPTION
**Pull request by stichbury**
## Description
PR to add "Hello Kedro" example to `kedro-examples` so we have an example file to link to in the docs

## Development notes
Added an example to illustrate basic concepts as described in the [documentation](https://github.com/quantumblacklabs/private-kedro/blob/develop/docs/source/02_getting_started/04_hello_kedro.md)

## Checklist

- [x ] Opened this PR as a 'Draft Pull Request' if it is work-in-progress
- [x ] Assigned myself to the PR
- [ ] Added tests to cover my changes

## Notice

- [ x] I acknowledge and agree that, by checking this box and clicking “Submit Pull Request”:

- I submit this contribution under the [Apache 2.0 license](https://www.apache.org/licenses/LICENSE-2.0.txt) and represent that I am entitled to do so on behalf of myself, my employer, or relevant third parties, as applicable.
- I certify that (a) this contribution is my original creation and / or (b) to the extent it is not my original creation, I am authorised to submit this contribution on behalf of the original creator(s) or their licensees.
- I certify that the use of this contribution as authorised by the Apache 2.0 license does not violate the intellectual property rights of anyone else.

### CODE REVIEW

File: hello-kedro/hello-kedro.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+
+def return_greeting():
+    # Prepare first node
+    return "Hello"
+
+
+return_greeting_node = node(
+    return_greeting, inputs=None, outputs="my_salutation"
+)
```
Comment (921kiyo) [inline hello-kedro/hello-kedro.py:16]:
```suggestion
return_greeting_node = node(return_greeting, inputs=None, outputs="my_salutation")
```

Comment (921kiyo) [reply hello-kedro/hello-kedro.py:16]: I ran black formatter.

File: hello-kedro/README.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# Hello Kedro Example
+
+This is the example code described in the [Hello Kedro](https://github.com/quantumblacklabs/private-kedro/blob/develop/docs/source/02_getting_started/04_hello_kedro.md) documentation.
```
Comment (921kiyo) [inline hello-kedro/README.md:3]:
```suggestion
This is the example code described in the [Hello Kedro](https://kedro.readthedocs.io/en/latest/02_getting_started/04_hello_kedro.md) documentation. 
```

File: hello-kedro/README.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+# Hello Kedro Example
+
+This is the example code described in the [Hello Kedro](https://github.com/quantumblacklabs/private-kedro/blob/develop/docs/source/02_getting_started/04_hello_kedro.md) documentation. 
+
+<!-- NOTE: LINK TO CHANGE SHORTLY -->
```
Comment (921kiyo) [inline hello-kedro/README.md:5]: What is this? Is it possible to remove this note, and apply the change with another PR?

File: hello-kedro/README.md (hunk 2 above)
Comment (921kiyo) [reply hello-kedro/README.md:3]: This link should be available once we release it :)

File: hello-kedro/hello-kedro.py (hunk 1 above)
Comment (laisbsc) [reply hello-kedro/hello-kedro.py:16]: I ran black too with length of 79 char. That line has 82, I think.

**Pull request merged**